fuse/ntfs-3g/Portfile

```diff
@@ -20,18 +20,19 @@ long_description    The NTFS-3G driver is an open source, freely available \
                     Also included are ntfsprogs, a set of utilities to create \
                     and manipulate NTFS file systems.
 
-homepage            http://www.tuxera.com/community/ntfs-3g-download/
+homepage            https://www.tuxera.com/company/open-source/
 distname            ntfs-3g_ntfsprogs-${version}
 extract.suffix      .tgz
 
-master_sites        http://tuxera.com/opensource/
+master_sites        https://tuxera.com/opensource/
 
-checksums           rmd160  48f76824ae38d1cef0f872290a44c2eaef576a29 \
-                    sha256  0489fbb6972581e1b417ab578d543f6ae522e7fa648c3c9b49c789510fd5eb93 \
-                    size    1318476
+checksums           rmd160  4f17aecd0a9e6ff1fd44e434d0484ac1e784ef1e \
+                    sha256  f20e36ee68074b845e3629e6bced4706ad053804cbaf062fbae60738f854170c \
+                    size    1324320
 
+livecheck.url       https://github.com/tuxera/ntfs-3g/releases
 livecheck.type      regex
-livecheck.regex     {stable version</span></b> is <a href="https://tuxera.com/opensource/ntfs-3g_ntfsprogs-(.+?)\.tgz"}
+livecheck.regex     {>Release (.+)</a>}
```

Comment:
Instead of coding a GitHub livecheck manually, could you use the github portgroup which will do it for you?

Reply:
I've modified the PR to use the `github` PortGroup.